<launch>
	<node pkg="performance_tests" type="cpp_talker" name="cpp_talker">
		<param name="hertz" type="int" value="1"/>
	</node>

	<node pkg="performance_tests" type="py_listener.py" name="py_listener"/>
</launch>
